### PR TITLE
[Upgrade py3] Futurize django-sendfile

### PR DESCRIPTION
**Pull request by bhaidaya**
Used python future library to auto migrate from Python 2 to Python 3.

https://github.com/enefeq/django-sendfile

### CODE REVIEW

File: sendfile/__init__.py [hunk 1]
```diff
@@ -10,7 +8,7 @@
 import unicodedata
 
 
-def _lazy_load(fn):
+def _lazy_load(fn):
```
Comment (msde) [inline sendfile/__init__.py]: trailing whitespace?

File: sendfile/__init__.py [hunk 2]
```diff
@@ -58,9 +56,9 @@ def sendfile(request, filename, attachment=False, attachment_filename=None, mime
     '''
 
     #TODO: PY3Upgrade - Cleanup
-    # Intentially ninja import future libraries into ridiculous __init__.py file
+    # Intentionally ninja import future libraries into ridiculous __init__.py file
     # (python 3 upgrade, py2/3 compatible)
-
+    from builtins import map
```
Comment (msde) [inline sendfile/__init__.py]: I don't think this function actually calls map anywhere, it's used on line 2 to stitch `VERSION` together into `__version__`

Comment (bhaidaya) [reply sendfile/__init__.py]: Woah. Thanks. Moving too fast.

**Comment by aisipos** [comment]
Closing for now. e3-server uses this PR with a pinned commit. Merging to master in this repo has little meaning, we had been using the `e3` branch. Medium / long term plan is to migrate to another library entirely.

**Pull request closed without merging**